README.rst

```diff
@@ -15,11 +15,56 @@ simphony-kratos is hosted on github: https://github.com/simphony/simphony-kratos
 Requirements
 ------------
 
-- somphony-common > 0.1.1
+- `simphony-common`_ == 0.1.5
+
+.. _simphony-common: https://github.com/simphony/simphony-common
 
 Installation
 ------------
 
+Installation is based on setuptools and requeires Python 2.7::
+
+    # build and install
+    python setup.py install
+
+or::
+
+    # build for in-place development
+    python setup.py develop
+
 Kratos Installation
--------------------
+~~~~~~~~~~~~~~~~~~~
+
+Both Kratos-CFD and DEMPack wrappers use KratosMultiphysics. A suitable version to run with these wrappers can be downloaded here:
+
+- `KratosMultiphysics`_
+
+.. _KratosMultiphysics: https://web.cimne.upc.edu/users/croig/data/kratos-simphony.tgz
+
+Once downloaded and extracted in $EXTRACT_DIR a couple of system variables need to be defined::
+
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$EXTRACT_DIR/kratos/libs
+    export PYTHONPATH=$PYTHONPATH:$EXTRACT_DIR/kratos
+    
+Usage
+-----
+
+To use any of the wrappers after the installation they need to be imported as plugins.
+
+For Kratos-CFD::
+
+  from simphony.engine import kratos
```

Comment:
Please update the documentation to reflect the plugin endpoints as they are setup by `setup.py`